parametric/run.sh

```diff
@@ -6,4 +6,33 @@ ARGS=$*
 # FIXME: have to ignore the root conftest as it does a bunch of initialization/teardown
 #        not required for the integration/shared tests.
 PARENT_DIR=$(dirname $PWD)
-exec python -m pytest -c $PWD/conftest.py $ARGS
+DEFAULT_N=auto
```

Comment:
could we make this name a bit more descriptive for what it does (for those who will be less familiar with pytest-split)

Reply:
Renamed according to the terminology from the documentation.